tests/Unit/inc/Engine/Optimization/DelayJs/Admin/Subscriber/addOptions.php

```diff
@@ -1,32 +0,0 @@
-<?php
```

Comment:
@engahmeds3ed Removed this test as it's not necessary, the method in the subscriber is directly calling the dependency method with no processing